.gitignore

```diff
@@ -26,3 +26,6 @@ node_modules
 # dist dir
 /dist
 /*.zip
+
+# fixtures generated by tape-nock
+fixtures
```

Comment:
Fixtures that are required to run the tests should be committed. Devs should not be required to have access to external APIs to validate this repo's code. If tape-nock is going to be used to create the fixtures, it should be used only when we are validating for a breaking change in the WMATA API. At that point, we would commit the updated fixtures.